CI_YAML.md

```diff
@@ -69,6 +69,23 @@ targets:
        ["framework", "hostonly"]
    postsubmit_properties:
      - upload_metrics: "true"
+
+#
+# Devicelab example:
+# For tests that are located https://github.com/flutter/flutter/tree/master/dev/devicelab/bin/tasks:
+# 1) a `taskname` property is required, which should match the task name
+# 2) target name follows format of `<platform> <taskname>`
```

Comment:
The numbering here is not logical to me. "1" refers to the last line of the segment, "2" to the first.

Reply:
Recordered.